app/controllers/flood_risk_engine/enrollments/steps_controller.rb

```diff
@@ -51,16 +52,17 @@ def step
       def step_url(options = {})
         enrollment_step_path(enrollment, enrollment.current_step, options)
       end
+      helper_method :step_url
```

Comment:
I need to remove this - it was used in an early version, but is no longer needed.